src/test/java/org/springframework/samples/petclinic/rest/VisitRestControllerTests.java

```diff
@@ -149,6 +155,9 @@ public void testGetAllVisitsSuccess() throws Exception {
             .andExpect(content().contentType("application/json"))
         	.andExpect(jsonPath("$.[0].id").value(2))
         	.andExpect(jsonPath("$.[0].description").value("rabies shot"))
+        	.andExpect(jsonPath("$.[0].adHoc").exists())
+        	.andExpect(jsonPath("$.[0].scheduled").exists())
+        	.andExpect(jsonPath("$.[0].isPaid").exists())
         	.andExpect(jsonPath("$.[1].id").value(3))
         	.andExpect(jsonPath("$.[1].description").value("neutered"));
     }
```

Comment:
There is no new test for the new API -> "markPaid".
Preferably we should have separate unit, API and integration/service tests.